src/cmdc_tools/datasets/official/MN/data.py

```diff
@@ -96,6 +96,13 @@ def get(self):
         out = df.melt(
             id_vars=["dt", "fips"], var_name="variable_name", value_name="value"
         )
+
+        # Make sure the values are integers by forcing a numeric
+        # convert and then dropping missing and setting as int
+        out["value"] = pd.to_numeric(out["value"], errors="coerce")
```

Comment:
I just merged a similar change to master!

chose to do `.dropna().assign(value=lambda x: x["value"].astype(int))`

Reply:
Merged master to this branch -- Let's wait for tests to clear, but, if they pass, I'll merge afterwards.